doc/man/zend.1

```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
-.TH ZEND "1" "September 2022" "zend v3.2.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.1.
+.TH ZEND "1" "January 2023" "zend v3.2.1-679b76dd4" "User Commands"
 .SH NAME
-zend \- manual page for zend v3.2.0
+zend \- manual page for zend v3.2.1-679b76dd4
 .SH DESCRIPTION
-Zen Daemon version v3.2.0
+Zen Daemon version v3.2.1\-679b76dd4
```

Comment:
See above.

Reply:
Addressed with force push (for keeping history clean) in 58c1f119cd7d8eae61fe426a91a562767fd76c70.